build/common/installer/scripts/tomlparser-prom-customconfig.rb

```diff
@@ -342,6 +346,13 @@ def populateSettingValuesFromConfigMap(parsedConfig)
                 file.close
                 puts "config::Successfully created telemetry file for prometheus sidecar"
               end
+            elsif !@os_type.nil? && !@os_type.empty? && @os_type.strip.casecmp("windows") == 0
+              # Setting the environment variable for starting telegraf only when monitorKubernetesPods is true 
+              file = File.open("setpromenv.ps1", "w")
+              if !file.nil?
+                commands = get_command_windows("TELEMETRY_CUSTOM_PROM_MONITOR_PODS", monitorKubernetesPods)
```

Comment:
can there be a scenario where monitorKubernetesPods is neither true or false (instead it is a malformed input). if it is possible then what will be outcome?